tests/bulk/bulkwrite-update-arrayFilters-001.phpt

```diff
@@ -2,12 +2,12 @@
 MongoDB\Driver\BulkWrite::update with arrayFilters
 --SKIPIF--
 <?php require __DIR__ . "/../utils/basic-skipif.inc"; ?>
-<?php NEEDS('STANDALONE'); CLEANUP(STANDALONE); ?>
+<?php START('THROWAWAY', ["version" => "36-release"]); CLEANUP(THROWAWAY); ?>
```

Comment:
This shouldn't be necessary, since 3.6 should be the default for MO (as the first version listed).